core/src/main/kotlin/entity/Attachment.kt

```diff
@@ -23,6 +24,16 @@ public data class Attachment(val data: AttachmentData, override val kord: Kord)
      */
     val filename: String get() = data.filename
 
+    /**
+     * The description for the file.
+     */
+    val description: Optional<String> get() = data.description
+
+    /**
+     * The attachment's [media type](https://en.wikipedia.org/wiki/Media_type).
+     */
+    val contentType: Optional<String> get() = data.contentType
```

Comment:
Use nullable types instead of `Optional` in core entities.
```suggestion
    /**
     * The description for the file.
     */
    val description: String? get() = data.description.value

    /**
     * The attachment's [media type](https://en.wikipedia.org/wiki/Media_type).
     */
    val contentType: String? get() = data.contentType.value
```